File: 0058-length-of-last-word/0058-length-of-last-word.c
int lengthOfLastWord(char* s) {
    int length = strlen(s);
    int i = length - 1;
    int wordLength = 0;
    while (i >= 0 && s[i] == ' ') {
        i--;
    }
    while (i >= 0 && s[i] != ' ') {
        wordLength++;
        i--;
    }
    return wordLength;
}

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int count=0;
        int index=s.length()-1;
        while(s.charAt(index)==' '){
            index--;
        }
        for(int i=index;i>=0;i--){
            if(s.charAt(i)==' '){
                break;
            }
            count++;
        }
        return count;
    }
}